README.md

```diff
@@ -1,132 +1,108 @@
+2. Copy `csv-importer.lke` file to the following path `<linkurious>/data/plugins`
+3. From the Linkurious Enterprise dashboard, go to `Admin` -> `Global configuration`
 4. Scroll to the `Plugin settings` field
-
-5. If there are no other plugins configured here, replace the whole content of the field with the following code and skip to step 7. \
-Otherwise, go to step 6
+5. On the `Plugin settings` section, modify the content as follows:
 
     ```json
     {
-        "importer": {
-            "basePath": "importer"
-        }
-    }
-    ```
-
-6. On the `Plugin settings` field, add an new empty line before the last `}`, add in this line a comma (`,`) and, after the comma, paste the following code:
-
-    ```json
-    "importer": {
+      "csv-importer": {
         "basePath": "importer"
+      }
     }
     ```
-
 7. Click on `Save`
 
-8. Done!
+## Access the plugin
 
-### 2.2 Setting the Custom Action
+To access the plugin it's recommended to create a custom action that will appears in right-click context menu within the Linkurious 
+Enterprise UI.
 
-1. Open the *Custom action* panel (more information [here](https://doc.linkurio.us/user-manual/latest/custom-actions/#managing-custom-actions))
-
-2. Click on `NEW CUSTOM ACTION`
+![](readme_assets/customaction.png)
 
-3. Fill the `Custom action name` field with the following text: `Import data` 
+In order to do so:
 
+1. Open the *Custom action* panel (more information [here](https://doc.linkurio.us/user-manual/latest/custom-actions/#managing-custom-actions))
+2. Click on `NEW CUSTOM ACTION`
+3. Give a title within the `Custom action name` field
 4. Fill the `URL template` field with the following URL: `{{baseurl}}plugins/importer/index.html?sourceKey={{sourcekey}}`
+5. (Optional) Share the Custom Action with all users
+6. Click on `SAVE`
 
-5. (Optional) Share the Custom Action with all the users
+> If you prefer to access the plugin directly via URL, you need to retrieve the sourceKey and replace it within the link:
+> https://<your-linkurious-instance>/plugins/importer/index.html?sourceKey=<sourceKey>
 
-6. Click on `SAVE`
+## Use the plugin
 
-7. Done!
+1. Choose your CSV file and click the Upload button
+2. Specify if you are uploading nodes or edges
+3. Confirm the category name/edge type and the list of properties
+7. For edges, specify the categories for the source and destination nodes
+8. Click import
 
-## 3. How to import data
+### Import status
 
-1. Open a visualization
+After the import, you will get one of the following results:
 
-2. Execute the `Import data` custom action
+1. Successful: All nodes / edges have been imported
 
-3. Select the *csv* file from your computer and select the checkbox if this contains an *header*
+2. Failed: Nothing has been imported
 
-4. Click on `Load this file`
+3. Incomplete: Some nodes/edges failed to be imported due to one of the following reasons:
```

Comment:
There is too much text here, making bold `Incomplete:` would help reading, and `b. Unexpected properties (in strict schema)` too